zipkin-web/src/main/resources/app/js/component_data/default.js

```diff
@@ -1,15 +1,41 @@
 import {component} from 'flight';
 import $ from 'jquery';
+import queryString from 'query-string';
+import chai from 'chai';
```

Comment:
chai is an assertion framework used for javascript unit tests, we'd never run it in production code but I've included it here temporarily to perform the deep-equals comparison.